### PR TITLE
Release/2.0.0

### PR DESCRIPTION
**Pull request by cornelfd**
Changelog:
Delete Objective-C example projects as Objective-C is no longer supported in V2.
Update the Swift example projects:
- Created a very basic Loyalty example project, using UIKit
- Added a more complex Transaction Stream example project, using SwiftUI, that also shows the cards that are being enrolled and verified (on the spot, but also, later, using the `verifyCard` function).

### CODE REVIEW

File: Fidel.xcframework/ios-arm64/Fidel.framework/Production.xcconfig [hunk 1]
```diff
@@ -0,0 +1,6 @@
+BASE_URL=api.fidel.uk/v1
+
+// This should be the key of the same Split.io environment used by the API the base URL that is set above
+// No need to set it yourself here; Will be set by a script, based on the value that you set in the .env file.
+// Details in this doc: https://fidelapi.atlassian.net/wiki/spaces/CL1/pages/2352087154/Getting+started+with+development+for+the+iOS+SDK
+SPLIT_CLIENT_KEY=tNWNypmbxljZwQDM1RnMmJ2Zh5WdqVGOwV2NwZDbtV3asVma
```
Comment (valllllll2000) [inline Fidel.xcframework/ios-arm64/Fidel.framework/Production.xcconfig:6]: Should this be in the repo?

File: Fidel.xcframework/ios-arm64_x86_64-simulator/Fidel.framework/Production.xcconfig [hunk 2]
```diff
@@ -0,0 +1,6 @@
+BASE_URL=api.fidel.uk/v1
+
+// This should be the key of the same Split.io environment used by the API the base URL that is set above
+// No need to set it yourself here; Will be set by a script, based on the value that you set in the .env file.
+// Details in this doc: https://fidelapi.atlassian.net/wiki/spaces/CL1/pages/2352087154/Getting+started+with+development+for+the+iOS+SDK
+SPLIT_CLIENT_KEY=tNWNypmbxljZwQDM1RnMmJ2Zh5WdqVGOwV2NwZDbtV3asVma
```
Comment (valllllll2000) [inline Fidel.xcframework/ios-arm64_x86_64-simulator/Fidel.framework/Production.xcconfig:6]: Same as for the one above, would there be a way to not have this in the public repo?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,382 +1,5 @@
...
-Features:
-
-- Added camera card scanning
-- Added API link card endpoints
-- Banner image customisation
-- Added country code picker
-- API tests
+The changelogs for the iOS SDK can now be found here:
+- [version 1](https://fidelapi.com/docs/select/sdks/ios/releases)
+- [version 2](https://fidelapi.com/docs/stream/sdks/ios/releases)
```
Comment (diazproduction) [inline CHANGELOG.md:5]: Isn't Select also going to use V2? Shouldn't the links be more "[Select Transactions]" and "[Transaction Stream]"?

Comment (cornelfd) [reply CHANGELOG.md]: Please check the latest version, @diazproduction 🙏

**Review by diazproduction — APPROVED** [review]

**Pull request merged**